### DISCUSSION TITLE
I haven't been able to install spritepacks from others' Google Drives and stuff, how-do?

### DISCUSSION THREAD

**Discussion by Gwen-the-pen** [Monika-After-Story/MonikaModDev#9365] [general - spritepacks]
I've seen a bunch of people who've installed custom spritepacks and completely changed their Monika. I've tried the same but she can't ever read them! So I was wondering if anyone has a guide somewhere. . .?

**Comment by Justformas** [comment]
There's a guide that also includes troubleshooting information here: https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#installing-sprite-packs

**Reply by Gwen-the-pen** [reply]
Thank you, but that's for the official MAS spritepacks; there are many that other users have made and I wanted to know how I'm supposed to install them without Monika telling me she can't read the file.

**Reply by Justformas** [reply]
Could you give a link to one of these custom spritepacks?
